app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -0,0 +1,276 @@
+# -*- encoding : utf-8 -*-
```

Comment:
I think it has to be copied manually because it's not part of the gem but a later addition. Anyway, removal results in error.